### PR TITLE
Python version up

### PR DESCRIPTION
**Pull request by kristinaNikolaevaa**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,9 +21,12 @@ RUN apt update && \
 # Install 18.x Node.js
     curl -fsSL https://deb.nodesource.com/setup_18.x | bash - && \
     apt install -y nodejs && \
-# Install py3.8 from deadsnakes repository and pip from standard ubuntu packages
+# Install py3.10 from deadsnakes repository and pip from standard ubuntu packages
     add-apt-repository ppa:deadsnakes/ppa && apt update && \
-    apt install -y python3.8 python3-pip
```
Comment (gigimon) [inline Dockerfile:26]: you don't install python3.8 but install symlinks by update-alternatives?

File: Dockerfile [hunk 2]
```diff
@@ -40,6 +43,7 @@ RUN apt install -y libxkbcommon0 \
     xvfb
 
 COPY ./deploy/requirements/* /opt/
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
```
Comment (gigimon) [inline Dockerfile:46]: why you install addition pip? If you installed it by

**Review by gigimon — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (kristinaNikolaevaa) [reply Dockerfile:26]: It look like python3.8 is already installed in ubuntu container

File: Dockerfile (hunk 2 above)
Comment (kristinaNikolaevaa) [reply Dockerfile:46]: I removed first pip installation
I get this error with it(when try install requirements):
ImportError: cannot import name 'html5lib' from 'pip._vendor' (/usr/lib/python3/dist-packages/pip/_vendor/__init__.py)

**Pull request merged**